### PR TITLE
Document rpc crate

### PR DESCRIPTION
**Pull request by Liamsi**
Part of #339 and should be included before 0.14.0 release: #344 

rendered: https://github.com/informalsystems/tendermint-rs/tree/ismail/rpc_readme/rpc

* [x] Referenced an issue explaining the need for the change
* [x] Updated all relevant documentation in docs
* [ ] Updated all code comments where relevant
* [ ] Wrote tests
* [ ] Updated CHANGES.md

### CODE REVIEW

File: rpc/README.md [hunk 1]
```diff
@@ -0,0 +1,26 @@
+# tendermint-rpc
+
+A rust implementation of the core types returned by a Tendermint node's RPC 
+endpoint. 
+These can be used to deserialize JSONRPC responses.
+All networking related features will be feature guarded to keep the dependencies small 
+in cases where only the core types are needed.
+
+Releases of this crate should be compatible to the same [tendermint] version as
+the [tendermint.rs] crate with the same version number.
```
Comment (xla) [inline rpc/README.md:10]:
```suggestion
the [tendermint-rs] crate with the same version number.    
```

Is this a typo? Also we should get tendermint.rs !!!

File: rpc/README.md [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+These can be used to deserialize JSONRPC responses.
+All networking related features will be feature guarded to keep the dependencies small 
+in cases where only the core types are needed.
+
+Releases of this crate should be compatible to the same [tendermint] version as
+the [tendermint.rs] crate with the same version number.    
+
+## The `client` Feature
+
+Additionally, this crate includes a RPC client implementation to query Tendermint RPC endpoints.
```
Comment (xla) [inline rpc/README.md:14]:
```suggestion
Additionally, this crate includes an RPC client implementation to query Tendermint RPC endpoints.
```

File: rpc/README.md (hunk 1 above)
Comment (Liamsi) [reply rpc/README.md:10]: image

Comment (romac) [reply rpc/README.md:10]: > Also we should get tendermint.rs !!!

@ebuchman

**Pull request merged**